.github/workflows/release.yml

```diff
@@ -24,6 +24,9 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: '^1.13.7'
+      - name: Setup Ginkgo Test Framework
+        run: |
+          go get -u github.com/onsi/ginkgo/ginkgo
```

Comment:
Don't you get those complaints that you should not use `go get` but rather `go install`?

Reply:
fixed